vars/createSctRunner.groovy

```diff
@@ -2,16 +2,28 @@
 
 def call(Map params, Integer test_duration, String region) {
     def cloud_provider = getCloudProviderFromBackend(params.backend)
+    def instance_type_arg = ""
+    if ( params.backend == "k8s-local-kind-aws" ) {
+        instance_type_arg = "--instance-type c5.xlarge"
+    } else if ( params.backend == "k8s-local-kind-gce" ) {
+        instance_type_arg = "--instance-type e2-standard-4"
+    }
 
     println(params)
     sh """
     #!/bin/bash
     set -xe
     env
 
-    if [[ "$cloud_provider" == "aws" || "$cloud_provider" == "gce" || "$cloud_provider" == "k8s-local-kind-aws" || "$cloud_provider" == "k8s-local-kind-gce" ]]; then
+    if [[ "$cloud_provider" == "aws" || "$cloud_provider" == "gce" ]]; then
```

Comment:
Does not make any sense, why do you set `instance_type_arg` if you are not using it.
Are we stop using sct runner for `k8s-local-kind-*` backends ?